src/Tools/CompilerPerfTests/run-perf.ps1

```diff
@@ -0,0 +1,28 @@
+﻿Set-Variable -Name LastExitCode 0
+Set-StrictMode -Version 2.0
+$ErrorActionPreference = "Stop"
+
+try
+{
+    $SlnDir = "$PSScriptRoot/../../../"
+    $BinariesPath = "$SlnDir/Binaries"
+
+    . "$SlnDir/build/scripts/build-utils.ps1"
```

Comment:
Note: this script will set variables equivalent to `$SlnDir` and `$BinariesPath` : `$repoDir` and `$binariesDir`. Don't care as much about `$slnDir` but prefer you use the `$binariesDir` variable. There are long term plans to move the `Binaries` directory to artifacts and will be easier if this script is using the defined variable.

Reply:
How could I find the script without first calculating `$SlnDir`?